### PR TITLE
METAL-991: Add script to set up CAPI e2e tests

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

File: 06_create_cluster.sh [hunk 1]
```diff
@@ -123,6 +123,20 @@ if [[ ! -z "${ENABLE_VIRTUAL_MEDIA_VIA_EXTERNAL_NETWORK}" ]]; then
     oc patch provisioning provisioning-configuration --type merge -p "{\"spec\":{\"virtualMediaViaExternalNetwork\":true}}"
 fi
 
+if [[ -n "${ENABLE_CAPI_E2E}" ]]; then
```
Comment (dtantsur) [inline 06_create_cluster.sh:126]: Could it be a small separate script that is invoked in the new job?

Comment (honza) [reply 06_create_cluster.sh]: Done

Comment (honza) [reply 06_create_cluster.sh]: ... now I'm wondering how we will call it from the job: https://github.com/openshift/release/pull/58839

Comment (honza) [reply 06_create_cluster.sh:126]: I added the env var back.

**Pull request merged**